browser/model/installable-item.js

```diff
@@ -72,6 +72,10 @@ class InstallableItem {
     return this.productVersion;
   }
 
+  getProductSize() {
```

Comment:
Removed the function as of now. Will use this for calculating total download size.